__mocks__/genInteractives.js

```diff
@@ -42,20 +46,29 @@ const interactiveElementsMap = {
   menuitem:[],
   option: [],
   select: [],
-  'table[role="grid"]': [{prop: 'role', value: 'grid'}],
-  'td[role="gridcell"]': [{prop: 'role', value: 'gridcell'}],
+  th: [],
   tr: [],
   textarea: [],
+  video: [],
```

Comment:
`<video>` is now recognized as an interactive element, which eliminates situations like this, where developers feel they need to add a role:

https://github.com/tootsuite/mastodon/pull/1651#discussion_r111453507